data/hai/hai outfits.txt

```diff
@@ -696,7 +696,7 @@ outfit `"Benga" Reverse Thruster`
 	"reverse flare sprite" "effect/atomic flare/tiny"
 		"frame rate" 14
 	"flare sound" "atomic tiny"
-	description "Considering them to be well worth the steep cost, many Hai freighter pilots install reverse thrusters such as these to supplement the often poor steering on larger Hai vessels."
+	description "Considering them to be well worth the steep cost, many Hai freighter pilots install Reverse Thrusters such as these to supplement the often poor steering on larger Hai vessels."
```

Comment:
I would have thought this was talking about reverse thrusters in general, and the capitalisation doesn't fit as well here.

Reply:
Ninja lol

Reply:
Operating on the same frequency hahaha

Reply:
I'm pretty sure I filtered out all of these before pushing, but apparently not. Damn.